tests/test_schema.py

```diff
@@ -1971,8 +1987,8 @@ class Meta:
 
     def test_multiple_nested_self_fields(self, user):
```

Comment:
Rename as `test_multiple_pluck_self_field` and move away from `Nested` tests?

(Nitpicking: I think `rels = fields.Pluck('self', 'name', many=True, attribute='relatives')` fits in one line.)